core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -898,6 +911,21 @@ public void ensureMediaInfoLoaded(@NonNull Playable media) {
         }
     };
 
+    @VisibleForTesting
+    public static boolean isInTimeRange(int from, int to, int current) {
```

Comment:
I think this method fits better into SleepTimerPreferences. PlaybackService is already such a huge class

Reply:
Done